src/main/resources/hudson/plugins/performance/PerformancePublisher/config.jelly

```diff
@@ -1,11 +1,10 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
-  <f:entry title="${%Performance report}" field="parsers">
-    <f:hetero-list name="parsers" hasHeader="true"
-                   descriptors="${descriptor.getParserDescriptors()}"
-                   items="${instance.parsers}"
-                   addCaption="${%Add a new report}"/>
-  </f:entry>
+
+   <f:entry title="${%Report files}" field="reportFiles">
```

Comment:
We should have clear explanation that it will autodetect format, to not scare people coming from old articles, docs, or just upgrading.